openstack/objectstorage/v1/objects/requests.go

```diff
@@ -273,18 +275,23 @@ type CopyOpts struct {
 	ContentEncoding    string `h:"Content-Encoding"`
 	ContentType        string `h:"Content-Type"`
 	Destination        string `h:"Destination" required:"true"`
+	ObjectVersionID    string `q:"version-id"`
 }
 
-// ToObjectCopyMap formats a CopyOpts into a map of headers.
-func (opts CopyOpts) ToObjectCopyMap() (map[string]string, error) {
```

Comment:
Renaming the function breaks backward compatibility unfortunately.
You may want to restore a `ToObjectCopyMap()` function that just calls `ToObjectCopyParams()` so that we can merge and release in the next minor version. Or we can wait a little bit and merge for the next major, up to you.

Reply:
ok, let me try to fix this.

Reply:
done, see the recent changes

Reply:
Perfect, thanks.